docs/dapps/index.md

```diff
@@ -30,8 +30,7 @@ Rent-seeking third parties can exploit this fact to extract funds from data feed
 
 [Oracle Extractable Value (OEV)](https://medium.com/api3/oracle-extractable-value-oev-13c1b6d53c5b) is a subset of MEV that oracles have priority in extracting by batching additional operations with their updates.
 Furthermore, instead of searching for such OEV opportunities themselves, oracles can auction off this privilege.
-Api3 holds transparent and permissionless auctions for OEV opportunities on OEV Network, and [pays](/dapps/oev-rewards/) 80% of the auction proceeds to the respective dApps.
-OEV Rewards serves as a new and sustainable revenue stream for dApps.
+Api3 holds transparent and permissionless auctions for OEV opportunities on OEV Network, and pays [OEV Rewards](/dapps/oev-rewards/) to the respective dApps. OEV Rewards serves as a new and sustainable revenue stream for dApps.
```

Comment:
A term is needed to refer to the money raised by the auctions and I've been using "auction proceeds" for that. I'm against the use of "OEV proceeds" because it conflicts with the existing term and it's not obvious what it means (for example, it could as well include the OEV searcher revenue).